doc/lsst.pipe.base/creating-a-pipeline.rst

```diff
@@ -0,0 +1,448 @@
+###################
+Creating a Pipeline
+###################
+
+**Note**
+This guide will assume some knowledge about
+`PipelineTasks`\ s, and so if you would like you can check out
```

Comment:
It should be single so that it links against the documentation for the python class, but since I had the extra s in there, it was not doing it properly.